extensions/wikia/AdEngine/src/ad-engine.utils.js

```diff
@@ -0,0 +1,25 @@
+import AdEngine from 'ad-engine/src/ad-engine';
+import AdSlot from 'ad-engine/src/models/ad-slot';
+import Context from 'ad-engine/src/services/context-service';
+import TemplateService from 'ad-engine/src/services/template-service';
+import { build } from 'ad-engine/src/video/vast-url-builder';
+import BigFancyAdBelow from 'ad-products/src/modules/templates/uap/big-fancy-ad-below';
+import config from './context';
+
+Context.extend(config);
+
+TemplateService.register(BigFancyAdBelow);
+
+let adEngine = new AdEngine(config);
+adEngine.init();
+
+function initializeSlot(params) {
```

Comment:
Can we run customAdLoader and inject our own SlotService?
https://github.com/Wikia/ad-engine/blob/dev/src/services/custom-ad-loader.js#L6

So the code will use TemplateService correctly and use proper slots. What do you think?